civis/ml/tests/test_model.py

```diff
@@ -39,6 +39,10 @@
 from civis.ml import _model
 
 
+LATEST_TRAIN_TEMPLATE = 10582
```

Comment:
I'm not sure I understand what this change is for. It seems like this is another constant that we'll have to remember to update.

Reply:
Tests should fail if we don't update it, which makes it much easier to remember. The problem I had was that we'd used `max(_PRED_TEMPLATES)` to infer what I've set here as `LATEST_TRAIN_TEMPLATE`. With the way I've implemented model registration, we're no longer guaranteed that this `max` operation gives us the most recent template for training jobs. I could build in logic which find the maximum of the keys in `_PRED_TEMPLATES` which aren't also in `REGISTRATION_TEMPLATES`. I started to be concerned that adding extra logic like that would make the tests more reliable. Do you think it's worth writing a `get_latest_train_template` function for the tests?

Reply:
Thanks for the clarification. I think this change is fine; an obvious constant we have to change is better than having to debug a less obvious helper function.